schema/json/schema-12.0.0.json

```diff
@@ -350,6 +350,24 @@
         "installedSize": {
           "type": "integer"
         },
+        "provides": {
+          "items": {
+            "type": "string"
+          },
+          "type": "array"
+        },
+        "depends": {
+          "items": {
+            "type": "string"
+          },
+          "type": "array"
+        },
+        "preDepends": {
+          "items": {
+            "type": "string"
+          },
+          "type": "array"
+        },
```

Comment:
indeed we do since there are additional structs being added but more importantly there are fields that have been split off based on use case (so the semantics of some types have changed).